app/models/user.rb

```diff
@@ -293,10 +293,36 @@ def teacher?
     given_lectures.any?
   end
 
-  # a user is an editor iff he/she is a course editor or lecture editor or
+  # a user is a teachable editor iff he/she is a course editor or lecture
+  # editor
+  def teachable_editor?
+    edited_courses.any? || edited_lectures.any?
+  end
+
+  def teachable_editor_or_teacher?
+    teachable_editor? || teacher?
+  end
+
+  def can_edit_teachables?
+    admin? || teachable_editor_or_teacher?
+  end
+
+  # if you are not a teacher of lecture or a module editor,
+  # but just an additional editor of some lecture, you
+  # will not be considered active if all of your edited lectures
+  # are too old
+  def active_teachable_editor?
+    return false unless can_edit_teachables?
+    return true if admin || course_editor? || teacher?
+
+    edited_lectures.select { |l| l.term.nil? || !stale? }
```

Comment:
This fails as there is no `stale?` method on the user. Use `!l.stale?` instead.

Reply:
Fixed in commit https://github.com/MaMpf-HD/mampf/pull/452/commits/6e28d5df8cb6e288d4964a7f0625a0fb34cc113d